tcms/report/views.py

```diff
@@ -442,10 +442,10 @@ def _report_data_context(self):
                 self._data.generate_status_matrix(build_ids)
             )
 
-            status_ids = (TestCaseRunStatus.id_failed(),)
+            status_ids = (TestCaseRunStatus.with_status_id(TestCaseRunStatus.FAILED),)
             failed_case_runs = self.read_case_runs(build_ids, status_ids)
 
-            status_ids = (TestCaseRunStatus.id_blocked(),)
+            status_ids = (TestCaseRunStatus.with_status_id(TestCaseRunStatus.BLOCKED),)
```

Comment:
You can also use `._status_to_id()` directly. Maybe update the name to not start with `_`. We don't need extra layers of methods wrapping other methods, wrapping objects.get